src/components/Pokemon.js

```diff
@@ -0,0 +1,42 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import './pokemon.css'
+
+function Pokemon({pokemoninfo}) {
+    const [pokemon, setPokemon] = useState({})
+
+    useEffect(() => {
+        async function fetchPokedex() {
+            try{
+                const result = await axios.get(`${pokemoninfo}`)
+                console.log(result)
+                setPokemon(result.data)
+            }catch(error){
+                console.error(error)
+            }
+        }
+        void fetchPokedex()
+    },[])
+    return (
+        <div className='pokemon-card'>
+            {pokemon.id > 0 &&
```

Comment:
Dan kan deze check ook iets simpeler
```suggestion
            {pokemon &&
```